packages/mdc-textfield/_mixins.scss

```diff
@@ -239,3 +263,36 @@
     }
   }
 }
+
+// Private Color Mixins
+
+@mixin mdc-text-field-bottom-line-color_($color) {
+  .mdc-text-field__input,
+  .mdc-text-field__input::placeholder {
```

Comment:
why do you need the ::placeholder? Maybe add a comment explaining this, since it is not obvious

Reply:
I definitely don't need it there. I need it on the label color mixin for the css only version uses a placeholder instead of a label.